README.md

```diff
@@ -30,10 +32,14 @@ func main() {
     var resource3 Resource2
 
     // Register the routes.
+    // Rember to pass the pointer of
```

Comment:
Please for better readability avoid using to many lines and writing more than 80 chars per line, thanks.

Reply:
Please give to me the reindent comment since is not clear for me how do you wan't it. Thanks!

Reply:
I meant to avoid using a new line each 5/6 words but instead use it when you reach circa 80 chars per line as in:

```go
    // Register the routes.
    // Remember to pass the pointer of the resource and not the resource itself
    // otherwise the server will not work!
```

Also please note that there were two typos: *Rember* instead of *Remember* and *resurce* instead of *resource*.